src/plugins/cordova-plugin-camera/app-host-handlers.js

```diff
@@ -5,9 +5,14 @@ module.exports = function (messages) {
         Camera: {
             takePicture: function (success, fail, args) {
                 messages.call('takePicture', args).then(function (result) {
-                    // 'result' should be {data: <ArrayBuffer>, type: <mimeType>}, from which we'll create a blob
-                    var blob = new Blob([result.data], {type: result.type});
-                    success(URL.createObjectURL(blob));
+                    if (args && args[1] === 0) {
```

Comment:
While we're here, should we support the 3rd destination type as well?

### Camera.DestinationType : <code>enum</code>
**Kind**: static enum property of <code>[Camera](#module_Camera)</code>  
**Properties**

| Name | Type | Default | Description |
| --- | --- | --- | --- |
| DATA_URL | <code>number</code> | <code>0</code> | Return base64 encoded string. DATA_URL can be very memory intensive and cause app crashes or out of memory errors. Use FILE_URI or NATIVE_URI if possible |
| FILE_URI | <code>number</code> | <code>1</code> | Return file uri (content://media/external/images/media/2 for Android) |
| NATIVE_URI | <code>number</code> | <code>2</code> | Return native uri (eg. asset-library://... for iOS) |